site/_apps/viper-news.txt

```diff
@@ -12,12 +12,12 @@ api: 'Hacker News Firebase API'
 hosting: 'Google App Engine'
 authors: 
     - name: 'Andrea'
-lighthouse: '93/100'
-interactive-em: '3.1s'
-interactive-faster-3g: '2.0s'
-lighthouse-link: https://www.webpagetest.org/lighthouse.php?test=170511_7Q_15882b2c52ed51c614ec707736e52940&run=3
-wpt-em-link: https://www.webpagetest.org/result/170511_AW_b2e2360d4207831ece22ab4210ab9444/
-wpt-faster-3g-link: https://www.webpagetest.org/result/170511_7Q_15882b2c52ed51c614ec707736e52940/
+lighthouse: '73/100'
+interactive-em: '2.4s'
+interactive-faster-3g: '1.7s'
+lighthouse-link: https://www.webpagetest.org/lighthouse.php?test=170611_DC_6bf5fb102603eb30b3d57dc4c25b480b&run=1
```

Comment:
No offense taken, I'm all up for formalizing these aspect of this PWA too, since the score is what most will look at the end.

And about that ... mind re-running through the latest changes?
https://www.webpagetest.org/result/170612_KX_b688363241070f0073415f46f2264ab2/